### ISSUE TITLE
Warnings for Gradle 4.5.1

### ISSUE THREAD

**Issue by alexei-osipov** [ksoichiro/gradle-console-reporter#8]
When executed with Gradle 4.5.1 produces this warnings message: 

```
> Task :jacocoTestReport
The TaskInternal.execute() method has been deprecated and is scheduled to be removed in Gradle 5.0. There are better ways to re-use task logic, see https://docs.gradle.org/4.5.1/userguide/custom_tasks.html#sec:reusing_task_logic.
```

**Comment by louth** [comment]
This really bugs me too - and I can't even see where an execute() method is used in the plugin code (just the tests). I'm not certain that @ksoichiro is still maintaining this it may need to be forked.

One workaround (albeit temporary) is to use gradle 4.7, add a gradle.properties file to root of the project and add:

``org.gradle.warning.mode=none`` 

Unfortunately this will only work until gradle 5.0.

**Comment by ksoichiro** [comment]
Sorry for my late reply.
I confirmed the issue. (#12 is related)
It seems the commit a9afddf (which has not been released) has already fixed the issue so I've just released v0.6.0.
